tests/test_client.py

```diff
@@ -911,28 +908,24 @@ def test_token_refresh_success(self):
             self.assertEqual(token_response, self.credentials.token_response)
 
     def test_recursive_authorize(self):
-        """Tests that OAuth2Credentials doesn't intro. new method constraints.
-
-        Formerly, OAuth2Credentials.authorize monkeypatched the request method
-        of its httplib2.Http argument with a wrapper annotated with
-        @util.positional(1). Since the original method has no such annotation,
-        that meant that the wrapper was violating the contract of the original
-        method by adding a new requirement to it. And in fact the wrapper
-        itself doesn't even respect that requirement. So before the removal of
-        the annotation, this test would fail.
-        """
+        # Tests that OAuth2Credentials doesn't intro. new method constraints.
```

Comment:
Note this isn't my content, though I'm happy to "you break it, you buy it".

I changed the docstring to a comment because docstrings on test cases make test runner (e.g. `nosetests`) harder to parse